### PR TITLE
created test for deletion of product

### LINKED ISSUE #8: TEST: Delete product
Write a new test in the `tests/product.py` module that verifies that a product can be deleted and not shown to users.

### PR DESCRIPTION
**Pull request by thejmdw**
Description of PR that completes issue here...

## Changes

- wrote test for the deletion of a product

## Testing

Description of how to test code...

- [ ] Run migrations
- [ ] Run test suite
- [ ] Seed database

- [ ] run `python3 manage.py test tests -v 1 ` in the command line

## Related Issues

- Fixes #8

### CODE REVIEW

File: tests/product.py [hunk 1]
```diff
@@ -96,5 +96,46 @@ def test_get_all_products(self):
...
+        data = { "id": 1 }
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+        response = self.client.delete(url, data, format='json')
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+        # GET GAME AGAIN TO VERIFY 404 response
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    # def test_delete_game(self):
```
Comment (hannahhall) [inline tests/product.py]: Can you delete this commented out function?

**Review by hannahhall — CHANGES_REQUESTED** [review]

File: tests/product.py (hunk 1 above)
Comment (thejmdw) [reply tests/product.py]: I removed it, thanks for catching that.

**Comment by thejmdw** [comment]
Hannah, I have removed the extra code.

**Review by hannahhall — APPROVED** [review]
👍

**Pull request merged**